src/contentScript.js

```diff
@@ -42,8 +86,9 @@ async function handleMarkup (wrapperEl, subreddit) {
 		md.classList.add('md');
 		const fakeBody = document.createElement('body');
 		fakeBody.classList.add('md-container');
+
 		// Avoid white background in Reddit dark mode
-		fakeBody.style.backgroundColor = 'inherit';
+		fakeBody.style.background = 'inherit';
```

Comment:
For my own knowledge, was this motivated by something specific? Does only setting `backgroundColor` cause issues with leftover background images or something?

Reply:
Yeah, r/madokamagica sets a background image on <body>.